ucm2/Intel/avs/avs_da7219/Google-Atlas-1.0-HiFi.conf

```diff
@@ -0,0 +1,67 @@
+SectionVerb {
+        EnableSequence [
+                cdev "hw:avsda7219"
+                cset "name='Headphone Jack Switch' off"
+                cset "name='Headset Mic Switch' off"
+                cset "name='Playback Digital Volume' 111"
+                cset "name='Playback Digital Switch' 1"
+                cset "name='Out DACL Mux' DAIL"
+                cset "name='Out DACR Mux' DAIR"
+                cset "name='Mixer Out FilterL DACL Switch' 1"
+                cset "name='Mixer Out FilterR DACR Switch' 1"
+                cset "name='ST Mixer Out FilterL Out FilterL Switch' 1"
+                cset "name='ST Mixer Out FilterR Out FilterR Switch' 1"
+                cset "name='Mic Volume' 5"
+                cset "name='Capture Digital Volume' 111"
+                cset "name='Capture Digital Switch' 1"
+                cset "name='Mixin Volume' 10"
+                cset "name='Mixin Switch' 1"
+                cset "name='Mixer In Mic Switch' 1"
+                cset "name='Out DAIL Mux' ADC"
+                cset "name='Out DAIR Mux' ADC"
+                cset "name='Gain Ramp Rate' 1"
+        ]
+        DisableSequence [
+        ]
+}
+
+SectionDevice."Headphone" {
+	Comment "Headphones"
+
+        Value {
+                PlaybackPCM "hw:avsda7219,1"
+                PlaybackMixerElem "Headphone"
+                JackDev "avs_da7219 Headset Jack"
+        }
+        EnableSequence [
+                cdev "hw:avsda7219"
```

Comment:
Does the above mean, that we can also remove cdev here?

Reply:
Yes, cdev lines are not required for one sound card. They are set by default.